docker/README.md

```diff
@@ -0,0 +1,3 @@
+Usage
+
+```docker build --no-cache=true -f docker/ubuntu/Dockerfile .```
```

Comment:
I also don't know. `--no-cache` was here before (the file was moved from `docker/hub/README.md` -> `docker/README.md` and I simply kept it unchanged